components/helm-broker/docs/proposals/addon-git-support.md

```diff
@@ -0,0 +1,163 @@
+# Support for loading Addons from Git repository
+
+Created on 2019-07-24 by Mateusz Szostok ([@mszostok](https://github.com/mszostok)).
+
+This document describes options for extending the ClusterAddonsConfiguration/AddonsConfiguration for supporting loading Addons from the Git repository.
+
+## Motivation
+
+Helm Broker fetches addons listed in the `index.yaml` file from remote HTTPS servers. Only zipped bundles can be fetched. This solution generates problems for end-user as it's not easy to always have dedicated HTTPS server which serves zipped bundles. A most common scenario is to have a git repository with unzipped bundles and index YAMLs. This document describes possible and accepted solutions for achieving that goal.
+
+## Accepted solution
+
+Git repositories can be specified by prefixing the address with the special `git::` prefix. After this prefix, any valid Git URL can be specified to select one of the protocols supported by Git. Inspired by the [Terraform implementation](https://www.terraform.io/docs/modules/sources.html#github).
+
+```yaml
+apiVersion: addons.kyma-project.io/v1alpha1
+kind: ClusterAddonsConfiguration
+metadata:
+  name: addons-cfg-sample
+spec:
+  repositories:
+    # Use SSH protocol with path to index YAML
+    - url: "git::git@github.com:kyma-project/bundles.git//index.yaml"
```

Comment:
I understand that https repositories, will have URLs as before? E.g. https://github.com/kyma-project/bundles/releases/download/0.6.0/index-testing.yaml

Reply:
exactly the https/http repos should be specified in the same way as previously. go-getter also supports that

proper example added